inspirehep/records/indexer/tasks.py

```diff
@@ -36,9 +36,9 @@ def get_record(uuid, record_version=None):
 
 
 @shared_task(ignore_result=False, bind=True)
-def batch_index(self, records_uuids, request_timeout):
+def batch_index(self, records_uuids, request_timeout=None):
     logger.info(f"Starting shared task `batch_index for {len(records_uuids)} records")
-    return InspireRecordIndexer().bulk_index(records_uuids)
+    return InspireRecordIndexer().bulk_index(records_uuids, request_timeout)
```

Comment:
``process_references_for_record`` which was using the ``batch_index`` doesn't have any tests that we have to update?

Reply:
Nope, it doesn't seem to be tested :(